docker/asreview-cli/Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM python:3.8
-RUN pip install asreview asreview-covid19 asreview-statistics asreview-visualization
+RUN pip install asreview asreview-statistics asreview-visualization
```

Comment:
Why is asreview-covid19 removed?

Reply:
Yes, support is still there and planned for v1. https://github.com/asreview/asreview-covid19/pull/41

Although we might want to reconsider this. I'm fine with archiving now that the dataset has become very big and COVID19 has less impact on daily life.

Reply:
We decided not to update the package anymore https://github.com/asreview/asreview-covid19/pull/46